make_dashboards/templates/external/js/dashboard.js

```diff
@@ -263,9 +263,9 @@ function make_throughput_plot(){
         });
     }
     // Subtitle text
-    var bp_per_week = total_count / num_weeks;
-    var mbp_per_second = (bp_per_week / (60*60*24*7))/1000000;
-    var subtitle_text = 'Average per week: '+parseInt(bp_per_week/1000000000)+' Gbp ('+parseInt(mbp_per_second)+' million base pairs per second)';
+    var bp_per_day = total_count / (num_weeks * 7);
+    var genomes_per_minute = (bp_per_day / (24*60)) / 3236336281;
```

Comment:
Should this be 1x coverage as it is now or 30x? I forgot what we talked about in the monday meeting 😂